frontend/src/views/c-summary.vue

```diff
@@ -201,7 +201,7 @@ export default {
       let totalCount = 0;
       this.repos.forEach((repo) => {
         repo.users.forEach((user) => {
-          if (user.checkedFileTypeContribution === undefined) {
+          if (user.checkedFileTypeContribution === -1) {
```

Comment:
I also think it might be more appropriate to replace `-1` with values such as `undefined` as default. Can you take a look again